buildprocess/configureWebpack.js

```diff
@@ -107,6 +107,15 @@ function configureWebpack(terriaJSBasePath, config, devMode, hot, MiniCssExtract
         ]
     });
 
+    config.module.rules.push({
+      test: /\.(ts|js)x?$/,
+      include: [path.resolve(terriaJSBasePath, "lib")],
+      use: {
+        loader: "istanbul-instrumenter-loader"
```

Comment:
`configureWebpack` is used to build TerriaMap, too, so won't this add instrumentation to apps, too? Or is that not a problem?

Reply:
Actually at the moment it's breaking the app build completely. See the Deployment log.

Reply:
I have made the configuration package local. Currently, it only reports classes referenced via the tests!